src/move/econia/sources/market.move

```diff
@@ -3470,7 +3469,7 @@ module econia::market {
         let market_order_id =
             ((order_book_ref_mut.counter as u128) << SHIFT_COUNTER);
         // If order eligible to post:
-        if (option::is_none(&cancel_reason_option)) {
+        if (option::is_none(&cancel_reason_option) && (remaining_size > 0)) {
```

Comment:
This combined with 

https://github.com/econia-labs/econia/blob/f5d5c133063bbe88735766e7800cffcaf95b6654/src/move/econia/sources/market.move#L3507-L3508

mean that any order that is cancelled here will have `remaining_size` set to zero. However, this shouldn't be the case. If, for instance, an IOC order partially filled and then cancelled due to the entire order not being immediately fillable, the remaining size should still not be zero as the order was not fully filled.

Reply:
@bogberry addressed as of f2c1777

Reply:
Ok, but in the future please link the commit in which the fix was made so I can look at the diff

Reply:
@bogberry will do, thanks for clarifying